src/app/services/auth.service.ts

```diff
@@ -122,15 +135,15 @@ export class AuthService {
     }
 
     public getSession() {
-        return {authenticated: this.authenticated, token: this.token, expires: this.expires};
+        return { authenticated: this.authenticated, token: this.token, expires: this.expires };
     }
 
     private fetchUserInfo() {
         if (this.token != null) {
             var headers = new Headers();
             headers.append('Authorization', `Bearer ${this.token}`);
             //noinspection TypeScriptUnresolvedFunction
-            this.http.get(this.oAuthUserUrl, {headers: headers})
+            this.http.get(this.oAuthUserUrl, { headers: headers })
```

Comment:
I would prefer that the formatting stay consistent with the rest of the code; no spaces inside the braces.